src/libs/events.ts

```diff
@@ -108,6 +113,7 @@ const eventsList = {
   workflowRerun: 'workflow:rerun',
   workflowUploadIO: 'workflow:uploadIO',
   workflowUseDefaultOutputs: 'workflow:useDefaultOutputs',
+  workflowsAppImport: 'workflowsApp:import',
```

Comment:
Should this event also be fired when importing from Dockstore?

Reply:
Do you mean when we use the featured workflows page? There's not really a difference between github and dockstore import from this page's point of view

Reply:
I mean on this page where users land when importing a workflow from Dockstore https://github.com/DataBiosphere/terra-ui/blob/37a1975d9827fe0e6b9bcc2a3abe4e46c9923cd2/src/pages/ImportWorkflow/ImportWorkflow.js#L71

Reply:
Good call, now added